src/components/Contentful/SectionWithBackgroundClassic.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+	<section class="">
+		<div class="tw-relative tw-p-0">
+			<div
+				class="tw-relative tw-w-full tw-overflow-hidden tw-z-1 tw-m-0 tw-top-0"
+				:class="verticalPaddingClasses"
+			>
+				<slot name="content">
+				</slot>
+			</div>
+			<div
+				class="tw-w-full tw-h-full tw-absolute tw-top-0 tw-z--1"
+				:style="backgroundStyle"
+			>
+				<video class="tw-w-full tw-h-full tw-object-cover" v-if="isBackgroundVideo"
+					:src="backgroundMedia.url"
+					autoplay
+					loop
+					muted
+					playsinline
+				></video>
+				<kv-contentful-img class="tw-w-full tw-h-full tw-object-cover section-with-background__media"
+					v-if="isBackgroundImage"
+					:width="1440"
+					:contentful-src="backgroundMedia.url"
+					fallback-format="jpg"
+					:alt="backgroundMedia.description"
+					:source-sizes="sourceSizes"
+				/>
+			</div>
+		</div>
+	</section>
+</template>
+
+<script>
+import KvContentfulImg from '@/components/Kv/KvContentfulImg';
+
+/**
+* Section Background
+* Creates a full width section with a full bleed background.
+* */
+
+export default {
+	components: {
+		KvContentfulImg,
+	},
+	props: {
+		/**
+		 * Content group content for background from Contentful type Background
+		* */
+		backgroundContent: {
+			type: Object,
+			default: () => {},
+		},
+		verticalPadding: {
+			type: Object,
+			default: () => {},
+		}
+	},
+	data() {
+		return {
+			sourceSizes: [
+				{
+					width: 1920,
+					height: 650,
+					media: 'min-width: 1441px',
+				},
+				{
+					width: 1440,
+					height: 620,
+					media: 'min-width: 1025px',
+				},
+				{
+					width: 1024,
+					height: 441,
+					media: 'min-width: 681px',
+				},
+				{
+					width: 680,
+					height: 372,
+					media: 'min-width: 481px',
+				},
+				{
+					width: 480,
+					height: 540,
+					media: 'min-width: 0px',
+				},
+			],
+		};
+	},
+	computed: {
+		backgroundStyle() {
+			if (this.backgroundContent?.backgroundColor) {
+				return {
+					'background-color': this.backgroundContent?.backgroundColor
+				};
+			}
+			return {};
+		},
+		isBackgroundVideo() {
+			return this.backgroundContent?.backgroundMedia?.file?.contentType.includes('video');
+		},
+		isBackgroundImage() {
+			return this.backgroundContent?.backgroundMedia?.file?.contentType.includes('image');
+		},
+		backgroundMedia() {
+			return {
+				description: this.backgroundContent?.backgroundMedia?.description ?? '',
+				title: this.backgroundContent?.backgroundMedia?.title ?? '',
+				url: this.backgroundContent?.backgroundMedia?.file?.url ?? ''
+			};
+		},
+		verticalPaddingClasses() {
```

Comment:
@ryan-ludwig @eddieferrer I'm not 100% sure these type of dynamic class generators are getting picked up with the tailwind compilation. _I know the dynamic number are not, but I'm not sure these defaults are either._

For this reason we may want to turn on the full set of padding classes based on our spacing scale, possibly also the grid options for work I'm about to start on.

Reply:
The defaults will get picked up.

Reply:
If I'm doing the math right it's 
28 spacing units * 4 breakpoints * 2 for top and bottom = 168 classes whitelisted for padding

Reply:
I don't have a better solution that works responsively off the top of my head though.

Reply:
Ok cool, I was seeing intermittent pick up of the defaults in localhost so happy to test that out in a real env.

I see 21 items in our space definitions but think we prob could cap that to the first 15 or so...

Reply:
Current setup adds 40 classes